path_config.go

```diff
@@ -64,12 +68,27 @@ func pathConfig(b *azureSecretBackend) *framework.Path {
 				Type:        framework.TypeBool,
 				Description: "Enable usage of the Microsoft Graph API over the deprecated Azure AD Graph API.",
 			},
+			"default_expiration": &framework.FieldSchema{
```

Comment:
Seeing "default" here, I expected that the rotate root could optionally accept an expiration (and we'd use the default if not provided). But there isn't any expiration parameter during rotation. Should there be? If not, then perhaps this shouldn't be called "default".